components/Header.js

```diff
@@ -19,6 +19,7 @@ const Header = () => {
     <header className="stickytop-0 bg-white justify-between">
       <div className="flex w-full p-6 items-center">
         <Image
+          alt=""
           src={
             "https://upload.wikimedia.org/wikipedia/commons/thumb/2/2f/Google_2015_logo.svg/2560px-Google_2015_logo.svg.png"
```

Comment:
do not put long strings into react code like this. setup a property file where you call pull these values like these and avoid mixing too much content into logic.

Reply:
also you should setup and follow a  specified max length of characters per line of code for all JS files in your project.